Build failure on Gentoo Linux due to missing include statement

Build of 2.7.1 fails on Gentoo Linux. See [Gentoo Bug Report 631252](https://bugs.gentoo.org/631252)

## Expected Behavior
Icinga compiles successfully.

## Current Behavior
Build fails at ```test/icinga-checkresult.cpp``` with:
```

test/icinga-checkresult.cpp: In function ‘void icinga_checkresult::NotificationHandler(const Ptr&, icinga::NotificationType)’:
test/icinga-checkresult.cpp:44:2: error: ‘cout’ is not a member of ‘std’
```

## Possible Solution
Add ```#include <iostream>``` to ```test/icinga-checkresult.cpp```.

--- a/test/icinga-checkresult.cpp
+++ b/test/icinga-checkresult.cpp
@@ -19,6 +19,7 @@
 
 #include "icinga/host.hpp"
 #include <BoostTestTargetConfig.h>
+#include <iostream>
 
 using namespace icinga;
 

